spec/javascript/components/applications/__snapshots__/ApplicationNewPage.test.js.snap

```diff
@@ -1194,7 +1194,6 @@ Array [
                                 onChange={[Function]}
                                 onFocus={[Function]}
                                 type="checkbox"
-                                value=""
```

Comment:
These values are removed from the snapshots due to default values being assumed by the Final Form fields